src/browser/api/notifications/observable_sequences.ts

```diff
@@ -33,18 +34,20 @@ export const runningTotal = Rx.Observable.merge(removesCounter, createsCounter)
     .scan((acc: number, value: number) => acc + value);
 
 export const noteStack = Rx.Observable.merge(
-    createdNotes.map((x: Identity) => {
+    createdNotes.map((x: NoteId) => {
         return {
             create: 1,
             name: x.name,
             uuid: x.uuid,
+            opacity: x.opacity
```

Comment:
Question, why does `opacity` go here and not simply into Window's options on creation in _javascript-adapter_?

Reply:
In the notifications there is an animation that overrides the window options that was previously hard-coded to change the notification opacity to 1 over one second when the notification was first shown on the screen (see genAnimationFunction fn in subscriptions.ts file).

Reply:
Got it, optional, but helpful would be to add a comment here explaining this